src/image-alt-text/auditEngine.js

```diff
@@ -85,6 +122,10 @@ export default class AuditEngine {
       return;
     }
 
+    const pageLanguage = getPageLanguage({ document: pageTags.dom?.window?.document });
+
+    this.log.info(`[${AUDIT_TYPE}]: Page language: ${pageLanguage}`);
```

Comment:
@igor-grubic if we leave this log in place, I think we should probably add which page it refers to. But maybe we don't need this since it will be in the audit's results?